tests/unit/build_tests/test_classgenerator.py

```diff
@@ -659,9 +658,6 @@ class TestGetClassSeparateNamespace(TestCase):
 
     def setUp(self):
         self.test_dir = tempfile.mkdtemp()
-        if os.path.exists(self.test_dir):  # start clean
```

Comment:
This code raised hidden errors during test setup. `tempfile.mkdtemp()` creates a clean temp directory so the below code is not needed